app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -728,31 +728,35 @@ def parse_base_item(item)
     def parse_image_name(image, image_ref)
       # parsing using same logic as in docker
       # https://github.com/docker/docker/blob/348f6529b71502b561aa493e250fd5be248da0d5/reference/reference.go#L174
-      parts = %r{
+      image_definition_re = %r{
         \A
+          (?<protocol>#{ContainerImage::DOCKER_PREFIXES.join('|')})?
           (?:(?:
             (?<host>([^\.:/]+\.)+[^\.:/]+)|
             (?:(?<host2>[^:/]+)(?::(?<port>\d+)))|
             (?<localhost>localhost)
           )/)?
           (?<name>(?:[^:/@]+/)*[^/:@]+)
-          (?:(?::(?<tag>.+))|(?:\@(?<digest>.+)))?
+          (?::(?<tag>[^:/@]+))?
+          (?:\@(?<digest>.+))?
```

Comment:
Can we have both a tag and a digest? (asking about the change from | to both parts being optional)